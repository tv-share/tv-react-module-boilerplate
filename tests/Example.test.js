import React from 'react';
import { Example } from "../src/index";
import renderer from 'react-test-renderer';

const emptyFn = () => {};

it('renders correctely', () => {
    const component = renderer.create(
        <Example
            label="Example"
            className="example-btn"
            onClick={emptyFn}
        />
    );
    
    expect(component.toJSON()).toMatchSnapshot();
});