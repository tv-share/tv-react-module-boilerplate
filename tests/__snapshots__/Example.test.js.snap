// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctely 1`] = `
<button
  className="example example-btn"
  onClick={[Function]}
>
  Example
</button>
`;
